tensorflow/lite/micro/kernels/fully_connected_test.cc

```diff
@@ -335,297 +141,789 @@ TfLiteStatus TestFullyConnectedQuantized(
+      F2Q(7, weights_min, weights_max), F2Q(8, weights_min, weights_max),
+      F2Q(9, weights_min, weights_max), F2Q(10, weights_min, weights_max),
+      F2Q(1, weights_min, weights_max), F2Q(2, weights_min, weights_max),
+      F2Q(3, weights_min, weights_max), F2Q(4, weights_min, weights_max),
+      F2Q(5, weights_min, weights_max), F2Q(6, weights_min, weights_max),
+      F2Q(7, weights_min, weights_max), F2Q(8, weights_min, weights_max),
+      F2Q(9, weights_min, weights_max), F2Q(10, weights_min, weights_max),
+      F2Q(1, weights_min, weights_max), F2Q(2, weights_min, weights_max),
+      F2Q(3, weights_min, weights_max), F2Q(4, weights_min, weights_max),
+      F2Q(5, weights_min, weights_max), F2Q(6, weights_min, weights_max),
+      F2Q(7, weights_min, weights_max), F2Q(8, weights_min, weights_max),
+      F2Q(9, weights_min, weights_max), F2Q(10, weights_min, weights_max),
+  };
+  const int bias_dims_data[] = {1, 3};
+  const int32_t bias_data[] = {
+      F2Q32(1, bias_scale),
+      F2Q32(2, bias_scale),
+      F2Q32(3, bias_scale),
+  };
+  const uint8_t expected_output_data[] = {
+      F2Q(24, output_min, output_max), F2Q(25, output_min, output_max),
+      F2Q(26, output_min, output_max), F2Q(58, output_min, output_max),
+      F2Q(59, output_min, output_max), F2Q(60, output_min, output_max),
+  };
+  const int output_dims_data[] = {2, 2, 3};
+
+  const int output_dims_count = 6;
+  uint8_t output_data[output_dims_count];
   TF_LITE_MICRO_EXPECT_EQ(
-      tflite::testing::TestFullyConnectedFloat(
-          tflite::testing::relu_input_dims, tflite::testing::relu_input_data,
-          tflite::testing::relu_weights_dims,
-          tflite::testing::relu_weights_data, tflite::testing::relu_bias_dims,
-          tflite::testing::relu_bias_data, tflite::testing::relu_golden,
-          tflite::testing::relu_output_dims, kTfLiteActRelu, output_data),
+      tflite::testing::TestFullyConnectedQuantized<uint8_t>(
+          input_dims_data, input_data, input_min, input_max, weights_dims_data,
+          weights_data, weights_min, weights_max, bias_dims_data, bias_data,
+          bias_scale, expected_output_data, output_dims_data, output_min,
+          output_max, kTfLiteActNone, output_data),
       kTfLiteOk);
 }
 
-TF_LITE_MICRO_TEST(SimpleTestQuantizedUInt8Relu) {
-  const float input_scale = 1.0f;
-  const int input_zero_point = 127;
-  const float weights_scale = 1.0f;
-  const int weights_zero_point = 128;
+// TODO(b/138811455): Fix code duplication in micro tests
+TF_LITE_MICRO_TEST(SimpleTestQuantizedInt8) {
+  using tflite::testing::F2Q32;
+  using tflite::testing::F2QS;
+
+  const float input_min = -63.5f;
+  const float input_max = 64.0f;
+  const float weights_min = -64.0f;
+  const float weights_max = 63.5f;
+  const float bias_scale = 0.25f;
+  const float output_min = -127.0f;
+  const float output_max = 128.0f;
+
+  const int input_dims_data[] = {2, 2, 10};
+  const int8_t input_data[] = {
```

Comment:
looks like an incorrectly resolved merge for fully_connected_test.cc

Reply:
oops, this PR should not be touching fully_connected_test.cc

fixed.